code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1262,3 +1268,27 @@
 		cell.charge = min(cell.charge + amount, cell.maxcharge)
 	if(repairs)
 		heal_bodypart_damage(repairs, repairs - 1)
+
+/mob/living/silicon/robot/proc/objectivesupdate()
+	var/mob/living/silicon/ai/AI
+	to_chat(src, "<span class='notice'>A.I Assigned Objective Updated. [AI.objectivesconfirm] Priority objective recieved as follows: [AI.objectiveupdate] </span>") //Objectives Updated, standby for Emergency Procedures. BEEEEP, BEEEP
+	busystatus = 1
+
+/mob/living/silicon/robot/verb/togglebusy()
```

Comment:
Give this some message to the cyborg. Currently, the cyborg gets no feedback but the ding.

Reply:
A message was included in the original commit, but I missed a Semicolon that was preventing Line 1283 from reading. Fixed.